os/include/mqueue.h

```diff
@@ -188,6 +188,13 @@ int mq_setattr(mqd_t mqdes, FAR const struct mq_attr *mq_stat, FAR struct mq_att
  * @since TizenRT v1.0
  */
 int mq_getattr(mqd_t mqdes, FAR struct mq_attr *mq_stat);
+/**
+ * @brief get message queue attributes
+ * @details @b #include <mqueue.h> \n
+ * POSIX API (refer to : http://pubs.opengroup.org/onlinepubs/9699919799/)
+ * @since TizenRT v1.0
+ */
+int mq_check(mqd_t mqdes);
```

Comment:
This looks like an internal api for mqueue, we need not expose it to outsize. Please check if we can we move this to some internal header file.